activesupport/lib/active_support/cache.rb

```diff
@@ -529,16 +529,17 @@ def expanded_key(key) # :nodoc:
 
         # Prefix a key with the namespace. Namespace and key will be delimited
         # with a colon.
-        def namespaced_key(key, options)
+        def normalize_key(key, options)
           key = expanded_key(key)
           namespace = options[:namespace] if options
           prefix = namespace.is_a?(Proc) ? namespace.call : namespace
           key = "#{prefix}:#{key}" if prefix
           key
         end
+        alias namespaced_key normalize_key
```

Comment:
This doesn't cover backwards compatibility. The file and mem cache stores changes break user written subclasses.